lib/charms/grafana_k8s/v0/grafana_dashboard.py

```diff
@@ -89,7 +118,8 @@ def __init__(
         self,
         charm: CharmBase,
         name: str,
-        event_relation: str = "monitoring",
+        event_relation: Optional[str] = "monitoring",
```

Comment:
Based on the code in this PR's diff it seems that `None` is not a valid value. Drop the Optional?
```suggestion
        event_relation = "monitoring",
```